.eslintrc.js

```diff
@@ -89,6 +88,7 @@ module.exports = {
     'github/no-inner-html': 'off',
     'github/role-supports-aria-props': 'off',
     'no-restricted-syntax': 'off',
+    'primer-react/a11y-use-next-tooltip': 'off',
```

Comment:
I disabled this because of having the source code of Tooltip v1 in this repo 🫠 We don't want eslint fixing our tooltip v1. 

As far as I have checked, we don't use any Tooltip v1 under the hood. So I think keeping this rule disabled until we remove the Tooltip v1 component is fine. Let me know if you think otherwise or if I missed anything!